packages/pointer/dev/index.tsx

```diff
@@ -1,4 +1,4 @@
-import { createPointerListeners, pointerHover, createPointerList } from "../src/index.js";
+import { createPointerListeners, pointerHover, createPointerList } from "../src/index-ols.js";
```

Comment:
This file does not exist.

Reply:
haven’t touched demo yet
well I did
but by mistake
it’s wip :)